NumberGuess/Program.cs

```diff
@@ -0,0 +1,142 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Text;
+using System.Threading.Tasks;
+
+namespace NumberGuess {
+    class Program {
+        static int maxPlayers = 2;
```

Comment:
Typically class fields would start with an uppercase letter like this: `MaxPlayers`.

Reply:
Fixed